src/containers/Interfaces/DyadCensus/useEdgeState.js

```diff
@@ -2,13 +2,13 @@ import { useState, useEffect } from 'react';
 import { actionCreators as sessionsActions } from '../../../ducks/modules/sessions';
 import { entityPrimaryKeyProperty } from '../../../ducks/modules/network';
 
-const getEdgeInNetwork = (edges, pair, edgeType) => {
+export const getEdgeInNetwork = (edges, pair, edgeType) => {
   if (!pair) { return null; }
   const [a, b] = pair;
 
   const edge = edges.find(({ from, to, type }) => (
     type === edgeType
-    && ((from === a && to === b) || (to === b && from === a))
+    && ((from === a && to === b) || (to === a && from === b))
```

Comment:
Ha! So it turns out to have been an implementation bug all along, and was intended to work the way it does not. Great find.